frontend/src/libs/components/dropdown/styles.module.scss

```
@@ -0,0 +1,97 @@
+@import "src/assets/css/vars.scss";
+
+$error-font-size: 12px;
+$line-height-coefficient: 1.54;
+$error-message-height: calc($error-font-size * $line-height-coefficient);
+
+.inputComponentWrapper {
+  display: flex;
+  flex-direction: column;
+  max-width: 327px;
+  color: $grey-dark;
+  font-weight: $font-weight-semibold;
+  font-size: 16px;
+  font-family: $font-family;
+  font-style: normal;
+  line-height: 140.5%;
+}
+
+.input {
+  width: 100%;
+  padding: 8px;
+  color: $grey-dark;
+  font-weight: $font-weight-regular;
+  font-size: 14px;
+  font-family: $font-family;
+  font-style: normal;
+  line-height: normal;
+  background: $white;
+  border: 2px solid $grey-light;
+  border-radius: 4px;
+  outline: none;
+}
+
+.input.filled {
+  color: $black;
+}
+
+.input:focus {
+  border-color: $blue;
+  box-shadow:
+    0 2px 4px -2px rgb(0 44 132 / 6%),
+    0 4px 8px -2px rgb(0 85 255 / 16%);
```

We have variables for the shadows.
```suggestion
  box-shadow: $shadow, $shadow-light;
```